sql/expression/in.go

```diff
@@ -310,11 +307,7 @@ func convertOrTruncate(ctx *sql.Context, i interface{}, t sql.Type) (interface{}
 }
 
 func (hit *HashInTuple) String() string {
-	pr := sql.NewTreePrinter()
-	_ = pr.WriteNode("HashIn")
-	children := []string{hit.Left().String(), hit.Right().String()}
-	_ = pr.WriteChildren(children...)
-	return pr.String()
+	return fmt.Sprintf("(%s HASH IN %s)", hit.Left(), hit.Right())
```

Comment:
This one you can leave alone if you want, it should never show up in something like `show create table`